### PR TITLE
Link Google Cloud courses by Lynn Langit

### PR DESCRIPTION
**Pull request by lynnlangit**
added link to Lynn Langit's GCP courses

### CODE REVIEW

**Comment by lynnlangit** [comment]
@googlebot I signed it!

File: README.md [hunk 1]
```diff
@@ -41,6 +41,7 @@ If you are new to Google Cloud Platform, there is a [free trial](https://cloud.g
 
 - ["The Google Cloud Developer Cheat Sheet" by Greg Wilson](https://github.com/gregsramblings/google-cloud-4-words) - A list of every product in the Google Cloud family described in 4 words or less.
 - ["Google Cloud - Jumpstart, Tutorials & Community!" by David das Neves](https://www.linkedin.com/pulse/google-cloud-jumpstart-tutorials-community-david-das-neves/) - Additional links about GCP including tutorials, solutions and community.
+- ["Google Cloud courses by Lynn Langit"](https://github.com/lynnlangit/learning-cloud/tree/master/GCP)- index link to 'GCP Essentials', 'gcp-for-bioinformatics', 'GCP ML' and more.  Lynn is the [first Google Cloud GDE (Developer Expert)](https://developers.google.com/community/experts/directory/profile/profile-lynn_langit).
```
Comment (tswast) [inline README.md:44]: Let's move these down to the other learning resources.

**Review by tswast — CHANGES_REQUESTED** [review]

**Review by rsamborski — APPROVED** [review]
Minor nits, otherwise good to merge.

**Comment by iennae** [comment]
Hey @lynnlangit 

Thank you for your contribution. As Tim requested, this link really should be moved towards the bottom especially as this is a list of links to other resources and not a direct link with supporting content on that page. Additionally the link to your GDE profile doesn't work anymore so should be updated/removed. 

Based on the age of this PR, I'm going to close this. Please feel free to reopen if you want to follow-up on the changes. 

Thank you, 

Jennifer

File: README.md (hunk 1 above)
Comment (lynnlangit) [reply README.md:44]: great - thanks

**Pull request closed without merging**